common/pb/command_roll_die.proto

```diff
@@ -5,4 +5,5 @@ message Command_RollDie {
         optional Command_RollDie ext = 1005;
     }
     optional uint32 sides = 1;
+    optional uint32 dice_to_roll = 2;
```

Comment:
I think the name "dice_to_roll" isn't a very clear identifier, why not just "amount" or "count"?

Reply:
also, might be nice to have a default = 1 here

Reply:
Defaults aren't allowed in proto3, so i'm gonna skip that. I feel the name is also correct as-is, as both "amount" and "count" are ambiguous.

Reply:
I disagree, "dice to roll" is ambiguous, it could be amount of dice, the fact that it is going to roll the dice is evident from the fact that it's in a command literally called roll die

Reply:
Fair enough. I'll move it to "count" then